lib/testing/src/lib/content-services/rest-apis/people/people-api-models.ts

```diff
@@ -0,0 +1,45 @@
+/*!
+ * @license
+ * Copyright 2019 Alfresco Software, Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+export interface PersonModel {
```

Comment:
exists a similar model in the js-api can we use it?

Reply:
this model is similar, but still different than what it is in js-api. I tried to use that one, but I don't get the same flexibility as with this model built strictly for the e2e simple needs (for example, some specific fields are required in Person from js-api, but in the e2e I don't want to enter those info every single time, and this model helps with simplicity: all you need to pass is the username)